x-pack/plugin/inference/src/main/java/org/elasticsearch/xpack/inference/external/response/cohere/CohereRankedResponseEntity.java

```diff
@@ -28,6 +28,10 @@
 
 public class CohereRankedResponseEntity {
 
+    private static final org.apache.logging.log4j.Logger logger = org.apache.logging.log4j.LogManager.getLogger(
```

Comment:
hmm I wonder if we can have this as an import like in other spots 🤔 so we don't have the class path

```
import org.apache.logging.log4j.LogManager;
import org.apache.logging.log4j.Logger;
```